src/Resources/views/Admin/app.html.twig

```diff
@@ -65,7 +65,17 @@
 
 <div class="uie-panels js-uie-panels">
     <div class="uie-panels__content js-uie-panels-selector">
-        <h2 class="ui header uie-heading uie-mb-md" id="uie-heading">{{ "Select an UI Element"|trans }}</h2>
+        <div class="uie-flex uie-flex-main-between uie-w-full uie-mb-md">
+            <h2 class="ui header uie-heading uie-m-0" id="uie-heading">
+                {{ "Select an UI Element"|trans }}
```

Comment:
@Guirec Do you think you could translate this as well by using a key instead of english words?

Reply:
Yep! I'll do that.